frontend_tests/node_tests/rendered_markdown.js

```diff
@@ -13,6 +13,7 @@ class Clipboard {
     constructor(...args) {
         clipboard_args = args;
     }
+    on() {}
 }
 
```

Comment:
You are on the right path here, you need to take two arguments for `on` here and call the second parameter I think

Reply:
Yeah, doing so worked. I can't believe I was just passing one argument to the ```on``` and wondering why it wasn't working. Thanks a lot brother, will update the other changes.